core/src/main/resources/error/error-classes.json

```diff
@@ -1520,6 +1525,25 @@
     ],
     "sqlState" : "42803"
   },
+  "MISSING_ATTRIBUTES" : {
+    "message" : [
+      "Operator missing attributes."
+    ],
+    "subClass" : {
+      "RESOLVED_ATTRIBUTE_APPEAR_IN_OPERATION" : {
+        "message" : [
+          "Resolved attribute(s) <missingAttributes> missing from <input> in operator <operator>.",
+          "Attribute(s) with the same name appear in the operation: <operation>.",
+          "Please check if the right attribute(s) are used."
+        ]
+      },
+      "RESOLVED_ATTRIBUTE_MISSING_FROM_INPUT" : {
+        "message" : [
+          "Resolved attribute(s) <missingAttributes> missing from <input> in operator <operator>."
```

Comment:
The both sub-classes have the same error format. Could you move this to its parent, and make the parent more specific.